screens/RequestScreen.js

```diff
@@ -70,30 +137,31 @@ export class RequestScreen extends Component {
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: "#0C94E1"
+    backgroundColor: '#0C94E1',
+    padding: 5,
   },
   contentContainer: {
-    justifyContent: "center",
-    paddingTop: 30
+    justifyContent: 'center',
+    paddingTop: 30,
   },
   input: {
     height: 40,
-    borderColor: "gray",
+    borderColor: 'gray',
     borderWidth: 1,
-    backgroundColor: "#fff",
+    backgroundColor: '#fff',
     marginBottom: 8,
-    padding: 5
+    padding: 5,
   },
   label: {
-    color: "#fff"
+    color: '#fff',
   },
   welcomeText: {
     fontSize: 30,
-    fontWeight: "bold",
-    color: "black",
+    fontWeight: 'bold',
+    color: 'black',
     lineHeight: 35,
     margin: 10,
     marginTop: 20,
-    textAlign: "center"
-  }
+    textAlign: 'center',
+  },
```

Comment:
oh boy, we're gonna have to figure out if we want single or double and configure prettier accordingly.

Reply:
I'll switch mine lol